packages/types/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@chainsafe/aleo-snap-types",
```

Comment:
Seems this package is better to be called shared, so can have code for the adapter and snap